app/helpers/host_aspects_helper.rb

```diff
@@ -0,0 +1,29 @@
+module HostAspectsHelper
+  # override host's tabs to add all aspects as tabs
+  def host_additional_tabs(host)
```

Comment:
Agreed, is `host_additional_views` better?
It returns hash of ids to template names.